macos/ONLYOFFICE/update/updates/ONLYOFFICE/changes/8.1.0/changes.html

```diff
@@ -91,46 +91,81 @@ <h2>New Features</h2>
 
         <h4>All Editors</h4>
         <ul>
-            <li>TODO</li>
+            <li>Switching to the creating forms in the <code>PDF</code> format</li>
+            <li>Added the ability to disable the 'Connect to cloud' section via the <code>--lock-portals</code> key</li>
+            <li>Added a window to restart the application if the settings marked with the asterisk are applied</li>
+            <li>Added the <code>Use graphics acceleration</code> option to the application settings</li>
+            <li>Added the ability to open a file by specifying its name in the terminal (Linux)</li>
```

Comment:
I'm not sure if this works on macOS, maybe you should delete the line

Reply:
it works, but not so usable. remove this line